### ISSUE TITLE
[SUGGESTION] | durotan go boat

### ISSUE THREAD

**Issue by KoiKarp** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1654]
**Describe your suggestion in full detail below:**

If Durotan decides to go south, he'd find the goblins in Booty bay, and could either choose to buy ships from them or try to steal ships, and then sail west into Pandaria, he'd land in Windward isle and gain the provinces of Sunglide and Windward

**Comment by arithon** [comment]
Would that lead to a Horde with Pandaren instead of Tauren? And Zandalari instead of Dark Spear? :D

**Comment by KoiKarp** [comment]
> Would that lead to a Horde with Pandaren instead of Tauren? And Zandalari instead of Dark Spear? :D

I don't think so as the Frostwolf Exodus isn't were the Horde goes

**Comment by arithon** [comment]
But Thrall/Go'el founded the second Horde :)

**Comment by KoiKarp** [comment]
> But Thrall/Go'el founded the second Horde :)

Yeah but not if the Frostwolves leave the Horde

**Comment by arithon** [comment]
> Yeah but not if the Frostwolves leave the Horde

Yes? Frostwolves ended up in Alterac lore-wise, no part of the Horde anymore. After the 2. War, Go'el found them and Lord of the Clans events took place...

**Comment by KoiKarp** [comment]
> > Yeah but not if the Frostwolves leave the Horde
> 
> Yes? Frostwolves ended up in Alterac lore-wise, no part of the Horde anymore. After the 2. War, Go'el found them and Lord of the Clans events took place...

Yes, but there is no mechanic for that in game?, every time i play as durotan Orgrim is always the leader of the horde after the Exodus

**Comment by arithon** [comment]
And after the first Horde fails, Thrall goes to Kalimdor and founds the new one, right?

**Comment by KoiKarp** [comment]
> And after the first Horde fails, Thrall goes to Kalimdor and founds the new one, right?

Sorry, what i meant is that if the Frostwolves leave the First Horde Thrall will not be the leader of the Second one

**Comment by zumbak04** [comment]
This suggestion sounds crazy... I like it.
